appconfig_0/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM deepview/linux-imx8:5.4.70
```

Comment:
This should be updated to use the maivin/debian (bookworm or previous bullseye tags)  container that we're supporting now, the deepview/linux-imx8 is deprecated.

https://hub.docker.com/repository/docker/maivin/debian/general

Reply:
Also most of the configuration below will no longer be required once you switch to maivin/debian as these are provided by the base Docker.  Refer to the GitHub repository for details.

https://github.com/MaivinAI/debian